simpa/core/device_digital_twins/pa_devices/ithera_msot_acuity.py

```diff
@@ -174,10 +164,32 @@ def update_settings_for_use_of_model_based_volume_creator(self, global_settings)
             Tags.STRUCTURE_TYPE: Tags.HORIZONTAL_LAYER_STRUCTURE
         })
 
-        volume_creator_settings[Tags.STRUCTURES]["mediprene"] = mediprene_layer_settings
+        # volume_creator_settings[Tags.STRUCTURES]["mediprene"] = mediprene_layer_settings
+
+        self.device_position_mm = np.add(self.device_position_mm, np.array([width_shift_for_structures_mm, 0, probe_size_mm]))
+        self.detection_geometry_position_vector = np.add(self.device_position_mm,
+                                                         np.array([0, 0,
+                                                                   self.probe_height_mm +
+                                                                   self.focus_in_field_of_view_mm]))
+        detection_geometry = CurvedArrayDetectionGeometry(pitch_mm=0.34,
+                                                          radius_mm=40,
+                                                          number_detector_elements=256,
+                                                          detector_element_width_mm=0.24,
+                                                          detector_element_length_mm=13,
+                                                          center_frequency_hz=3.96e6,
+                                                          bandwidth_percent=55,
+                                                          sampling_frequency_mhz=40,
+                                                          angular_origin_offset=np.pi,
+                                                          device_position_mm=self.detection_geometry_position_vector,
+                                                          field_of_view_extent_mm=self.field_of_view_extent_mm)
+
+        self.set_detection_geometry(detection_geometry)
+        for illumination_geom in self.illumination_geometries:
+            illumination_geom.device_position_mm = np.add(illumination_geom.device_position_mm,
+                                                          np.array([0, 0, probe_size_mm]))
 
         background_settings = Settings({
-            Tags.MOLECULE_COMPOSITION: TISSUE_LIBRARY.heavy_water(),
+            Tags.MOLECULE_COMPOSITION: TISSUE_LIBRARY.constant(mua=0.1),
```

Comment:
0.1 cm^-1 absorption is super high for the inside of the transducer head, no?

Reply:
Oh right, I changed this for testing purposes. I reverted it back :sweat_smile: